sites/svelte.dev/package.json

```
@@ -23,7 +23,8 @@
 		"do-not-zip": "^1.0.0",
 		"flexsearch": "^0.7.31",
 		"flru": "^1.0.2",
-		"sourcemap-codec": "^1.4.8"
+		"sourcemap-codec": "^1.4.8",
+		"svelte-local-storage-store": "^0.4.0"
```

it doesn't look like `svelte-local-storage-store` is being used in the diff?
```suggestion
		"sourcemap-codec": "^1.4.8"
```